Cargo.toml

```diff
@@ -9,39 +9,68 @@ members = [
     "bench"
 ]
 
+[workspace.dependencies]
+tokio = { version = "1.27.0", features = ["full"]}
+solana-sdk= { path = "../solana/sdk" }
```

Comment:
should be pointing to a github repo like master  branch or crate.

Reply:
Fixed